crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -4216,8 +4223,8 @@ mod tests {
             "src/a.py",
             &[
                 "Cannot divide object of type 'Literal[1]' by zero.",
-                "Cannot divide object of type 'Literal[2]' by zero.",
-                "Cannot divide object of type 'Literal[3]' by zero.",
+                "Cannot floor divide object of type 'Literal[2]' by zero.",
+                "Cannot reduce object of type 'Literal[3]' modulo zero.",
```

Comment:
This is the most consistent language I can find for this online.

Reply:
could possibly do something like this, but it's annoyingly verbose:

```suggestion
                "Using modulo with a zero divisor is undefined on an object of type 'Literal[3]'.",
```

Reply:
Yeah that seems too verbose to me and differs quite a bit from the other messages.

Reply:
I think what you have now is fine!